cocotb_test/simulator.py

```diff
@@ -832,6 +831,110 @@ def build_command(self):
         return [["vsimsa"] + ["-do"] + ["do"] + [do_file.name]]
 
 
+class Activehdl(Simulator):
+    def get_include_commands(self, includes):
+        include_cmd = []
+        for dir in includes:
+            include_cmd.append("+incdir+" + as_tcl_value(dir))
+
+        return include_cmd
+
+    def get_define_commands(self, defines):
+        defines_cmd = []
+        for define in defines:
+            defines_cmd.append("+define+" + as_tcl_value(define))
+
+        return defines_cmd
+
+    def get_parameter_commands(self, parameters):
+        parameters_cmd = []
+        for name, value in parameters.items():
+            parameters_cmd.append("-g" + name + "=" + str(value))
+
+        return parameters_cmd
+
+
+    def build_command_compile(self):
```

Comment:
Is OK but could we call this `build_script_sim`?

Reply:
You mean like in this  ?